source/operations/checklists/hardware.rst

```diff
@@ -314,8 +314,11 @@ Storage
             deviceName="$(basename $i)"
             echo "Modifying xfs max_retries and retry_timeout_seconds for drive $i mounted at $mountPath"
             echo 0 > /sys/fs/xfs/$deviceName/error/metadata/EIO/max_retries
+            echo 5 > /sys/fs/xfs/$deviceName/error/metadata/EIO/retry_timeout_seconds
```

Comment:
@balamurugana any reason we shouldn't pull this over to our general guidance? e.g. something specific directpv is doing where it might cause an issue on other xfs drives?

Reply:
Actually it feels somewhat extraneous - if max_retries is 0, then there's no point in setting retry_timeout_seconds right?

Reply:
I can do a PR in directpv then.

Reply:
@ravindk89 I couldn't able to understand the question.  What is other xfs drives? DirectPV mounts device with those tunings automatically. Refer https://github.com/minio/directpv/blob/master/pkg/xfs/mount_linux.go#L30

These tunings were suggested by @harshavardhana